eth/backend.go

```diff
@@ -222,14 +221,16 @@ func New(stack *node.Node, config *ethconfig.Config) (*Ethereum, error) {
 	if config.BlobPool.Datadir != "" {
```

Comment:
Also needs to be removed.

Reply:
Don't need, if `blob pool` is closed this variable won't be used. And keep changes to a minimum.

Reply:
Variables that are not used do not need to be loaded.
It should be included as a part of the disable blob changes.